dotnet/CustomerServiceAgent/README.md

```diff
@@ -0,0 +1,253 @@
+        Dashboard[Aspire Dashboard<br/>Traces, Logs, Metrics]
+    end
+    
+    subgraph "Application Services"
+        Orchestrator[Agent Orchestrator API<br/>CustomerServiceController]
+        OrderAPI[Order Service API<br/>In-memory order store]
+        CrmAPI[CRM Service API<br/>In-memory customer history]
+        ShippingAPI[Shipping Service API<br/>In-memory delivery management]
+        EmailAPI[Email Service API<br/>Mock email sender]
+    end
+    
+    subgraph "External Services"
+        EntraID[Microsoft Entra ID<br/>Token Acquisition & Validation]
+    end
+    
+    AppHost -->|Configures & Starts| Orchestrator
+    AppHost -->|Configures & Starts| OrderAPI
+    AppHost -->|Configures & Starts| CrmAPI
+    AppHost -->|Configures & Starts| ShippingAPI
+    AppHost -->|Configures & Starts| EmailAPI
+    
+    Dashboard -.->|Observes Telemetry| Orchestrator
+    Dashboard -.->|Observes Telemetry| OrderAPI
+    Dashboard -.->|Observes Telemetry| CrmAPI
+    Dashboard -.->|Observes Telemetry| ShippingAPI
+    Dashboard -.->|Observes Telemetry| EmailAPI
+    
+    Orchestrator -->|Service Discovery| OrderAPI
+    Orchestrator -->|Service Discovery| CrmAPI
+    Orchestrator -->|Service Discovery| ShippingAPI
+    Orchestrator -->|Service Discovery| EmailAPI
+    
+    Orchestrator -->|Acquire Token| EntraID
+    OrderAPI -->|Validate Token| EntraID
+    CrmAPI -->|Validate Token| EntraID
+    ShippingAPI -->|Validate Token| EntraID
+    EmailAPI -->|Validate Token| EntraID
+    
+    style AppHost fill:#512BD4,color:#fff
+    style Dashboard fill:#512BD4,color:#fff
+    style Orchestrator fill:#0078D4,color:#fff
+    style EntraID fill:#00BCF2,color:#000
+```
+
+## 🚀 Quick Start
+
+### Prerequisites
+
+- [.NET 9 SDK](https://dotnet.microsoft.com/download/dotnet/9.0)
+- [Visual Studio 2022](https://visualstudio.microsoft.com/) or [VS Code](https://code.visualstudio.com/) with C# extension
+- *(Optional)* [Microsoft 365 Developer account](https://developer.microsoft.com/microsoft-365/dev-program) for Graph API integration
+
+### Run the Sample
+
+1. **Clone the repository**
+   ```bash
+   git clone https://github.com/Azure-Samples/ms-identity-agent-identities.git
+   cd ms-identity-agent-identities/dotnet/CustomerServiceAgent
+   ```
+
+2. **Build the solution**
+   ```bash
+   dotnet build
+   ```
+
+3. **Run with .NET Aspire**
+   ```bash
+   dotnet run --project src/CustomerServiceAgent.AppHost
+   ```
+
+4. **Access the Aspire Dashboard**
+   - Open your browser to `https://localhost:15888` (or the URL shown in the console)
+   - Explore the service map, traces, logs, and metrics
+
+5. **Test the orchestration**
+   ```bash
+   # Using curl
+   curl -X POST https://localhost:7000/api/customerservice/process \
+     -H "Content-Type: application/json" \
+     -d '{"orderId": "12345", "userUpn": "agent@contoso.com"}'
+   
+   # Or use the .http file in VS Code
+   # Open: src/AgentOrchestrator/AgentOrchestrator.http
+   ```
+
```

Comment:
```suggestion
Note that this endpoint is on purpose anonymous, so that you can more easily test things out. In production you would need to uncomment the [Authorize] attribute on the endpoint.
```